Gemfile

```diff
@@ -66,3 +65,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem "pg", "~> 1.0"
+
+gem "devise_token_auth", "~> 0.2.0"
+
+gem "devise", "~> 4.4"
```

Comment:
did you have to change the devise version for something in particular?

Reply:
Yeah devise token auth didn't like the other one... So I uninstall them both, and reinstall first token auth and then devise...